### ISSUE TITLE
Stream doesnt start

### ISSUE THREAD

**Issue by mrwsl** [JohnCiubuc/StreamLinkerino#76]
Really cool project and it is exactly what I was looking for.
I built the testing branch without the docker step on arch (installed unpatched chatterino2 via AUR). When I start streamlinkerino, it opens another chatterino window and doesn't display the stream. Do I need to manually start it somehow?

**Comment by JohnCiubuc** [comment]
the project is sort-of not supported right now, but i'll try what I can. 

So the correct functionality is for the application to launch, then it'll launch chatterino. Then it'll merge chatterino into the app via some Xorg magic (therefore wonky if you're on wayland), and you should be good to go. If on wayland, most of the time the chatterino app will be launched but not merged, so you'll have it separate to the main application.

Whenever you switch tabs in chatterino, it can take upwards to around 20-30s for chatterino to write its file (located ~/.local/share/chatterino/Settings/window-layout.json). If that file doesn't exist, it would explain why streamlinkerino won't load/change mpv to the new channel. 

If your chatterino's window layout json is in a different location, lemme know so that I can update it. Also make sure that you have mpv installed as that's what the application uses to show its streams

**Comment by Wallboy** [comment]
Hi, I've just started using Streamlink and mpv and was looking for some sort of chat integration as well, and stumbled upon this project. 

Instead of creating a new Issue, I thought it'd be better to just ask here, since you mentioned this project is not really supported atm. I also looked at the first party Chatterino integration, and see no updates on that front since January. So I'm wondering if this project is just in limbo at the moment waiting on the merge requests in chatterino? Any chance to get this project (or the chatterino integration) as an AUR package for Arch Linux?

I'm glad I found there is projects working on getting Streamlink/mpv/twitch-chat all working as one unit. I hope this or the chatterino2 first party integration are completed! :)

**Comment by JohnCiubuc** [comment]
Thanks for the comment wallboy. This project is independent from the main chatterino update, though also just mainly waiting for chatterino first party integration to take the lead.

Main reason project is in limbo is that I do not have the time outside of medschool and other responsibilities to actively update this project. This project does work on wayland and X, tested on both, just have to launch the app outside of qtcreator on wayland on the very least. Though I am having some issues with chatterino launching at times, so I'll see if I can address that.

At the very least, as long as mpv launches and embeds, you can launch chatterino separately and it'll still read the file just fine (testing branch). Would love to push this on aur, but got to make sure it works fine at least 😅. Preferably i'd like to get a flatpak or appimage for this up, but /shrugs
